dev/clj_deconstruct_sigs/dev/data.clj

```diff
@@ -1,129 +0,0 @@
-(ns clj-deconstruct-sigs.dev.data
```

Comment:
I think if this data has no compatibility then deleting the ns is fine, but I would appreciate where/how `dev-resources/COSMIC_v3.2_SBS_GRCh38.txt` is obtained.

Reply:
Yes. I remove this snippet because it no longer works correctly. 

`dev-resources/COSMIC_v3.2_SBS_GRCh38.txt` is available from the official website (https://cancer.sanger.ac.uk/signatures/downloads/), and you can load it using `clj-deconstruct-sigs.db`.